Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -337,7 +337,7 @@ public bool IsFocused
 		[TypeConverter(typeof(VisibilityConverter))]
 		public bool IsVisible
 		{
-			get { return (bool)GetValue(IsVisibleProperty); }
+			get { return IsVisibleProperty != null ? (bool)GetValue(IsVisibleProperty) : false; }
```

Comment:
that can't be null.

Reply:
because if it was null, you'd get a `ArgumentNullException` instead

Reply:
:-(

Reply:
Stephane: Do you have any idea on how to solve this bug ?